src/plugins/intel_npu/src/al/include/intel_npu/icompiler.hpp

```diff
@@ -184,12 +180,6 @@ struct CompiledNetwork {
  */
 class ICompiler : public std::enable_shared_from_this<ICompiler> {
```

Comment:
CompiledNetwork should be removed from this header as well. It is a "Plugin owned compiled network storage that is required in case of a driver that doesn't support graph extension 1.7, as in this case plugin must create a copy of the compiled network".
The is no place for something like this in iCompiler.hpp anymore.

Reply:
done